src/main/scala/beam/router/BeamRouter.scala

```diff
@@ -431,7 +429,7 @@ object BeamRouter {
     leg: BeamLeg,
     vehicleId: Id[Vehicle],
     vehicleTypeId: Id[BeamVehicleType],
-    id: Int = UUID.randomUUID().hashCode(),
+    requestId: Int,
```

Comment:
Originally I used Singleton(https://github.com/LBNL-UCB-STI/beam/commit/cf31508fdd682678a0df0ceec99811ea76f19b69), but decided to refactor and inject it all the way for the future purpose (if we would change return type or make it distributed). But if it over engineering, I can rollback to Singleton

Reply:
Ok, switching back to Singleton because it breaks more than 10 tests and feel too lazy to fix `BeamService` dependency in all broken tests. Will leave this problem for the future version of us :)

Reply:
I agree because we don't know what the future version will need, and this minimizes the spiderwebs to one main class and a few usages